WooCommerce/Classes/System/SessionManager.swift

```diff
@@ -189,5 +217,37 @@ private extension SessionManager {
 
         keychain[username] = nil
         defaults[.defaultUsername] = nil
+        defaults[.defaultCredentialsType] = nil
+    }
+}
+
+// MARK: - Helpers to read `Credentials`
+//
+private extension Credentials {
```

Comment:
Should we make this extension public from `Credentials` to be reused in all places?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8446/commits/3c417e6644e4bbb34b789581b253087213fa1e52